.github/workflows/python-package.yml

```diff
@@ -0,0 +1,59 @@
+name: q2-umap CI
+
+on: [ push, pull_request ]
```

Comment:
Do you really need these 2? The tests are basically the same for both, no?

Reply:
I frequently push to branches before opening PR's and I find it nice CI is run on branches without an open PR.